editor/src/editor/layout/inspector/material/components/transparency.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from "react";
+import { EditorInspectorListField } from "../../fields/list";
+import { Material, StandardMaterial, PBRMaterial } from "babylonjs";
+
+
+export interface IEditorTransparencyModeFieldProps {
+	object: StandardMaterial | PBRMaterial;
+	onChange?: () => void;
+}
+
+export function EditorTransparencyModeField(props: IEditorTransparencyModeFieldProps): ReactNode {
+	const getTransparencyModeItems = () => {
```

Comment:
This can now be a const, or passed directly in the EditorInspectorListField component so we avoid function call.
That's all and then we can merge :)